### DISCUSSION TITLE
Project depleting its Disk IO Budget

### DISCUSSION THREAD

**Discussion by MODDEVBR** [supabase/supabase#20283] [Questions]
We are currently experiencing a critical system issue where our production system has become unresponsive. We have received notifications indicating that our project "[PROJECT]" is depleting its Disk IO Budget, even though our Disk IO Bandwidth consumption appears to be below 1% per day.

![image](https://github.com/supabase/supabase/assets/94724636/d20e2beb-b908-4c2e-a737-bde88f23d819)

We are reaching out to the community for assistance in resolving this problem. Our plan has the Spend cap enabled, and we are considering switching it to disabled temporarily to get the system up and running again. However, we want to ensure that we will not incur unnecessary charges for resources we are not actively using.

Any insights or assistance from the community would be greatly appreciated. Thank you in advance for your help.

**Comment by awalias** [comment]
Hey! It looks like the IO maxxed out to 100% on the 9th Jan, did anything happen on that day? A large amount of traffic or db requests? You should be able to debug in the logs explorer inside the dashboard if you're unsure where the requests are coming from https://supabase.com/dashboard/project/_/logs/explorer.

**Reply by maxwhalen** [reply]
It looks like it says 1.00% right? I am also having the same issue and have had a very minimal amount of requests (I think mine also says 1% of capacity has been used as far back as the project has existed).

**Comment by maxwhalen** [comment]
Following, I am having the same issue as of today

**Comment by Arkhemis** [comment]
Following as well, my IO consumption is at 1% and we're in a test environment with 3 users. Did you find out the root cause @maxwhalen ?
